tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -70,66 +70,61 @@ Feature: Sharing files and folders with internal users
       | read                | Custom role | Viewer        | read                      |
       | all                 | Custom role | Editor        | all                       |
 
-  @skip @yetToImplement
   Scenario: share a file with another internal user who overwrites and unshares the file
     Given user "user2" has logged in using the webUI
     When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
-    And the user shares file "new-lorem.txt" with user "User One" using the webUI
+    And the user shares file "new-lorem.txt" with user "User One" as "Editor" using the webUI
```

Comment:
we have sharing tests higher up, lets do this sharing as a Given step in the API